mempool/src/shared_mempool/coordinator.rs

```diff
@@ -158,18 +158,20 @@ pub(crate) async fn coordinator<V>(
                                 };
                             }
                             Event::RpcRequest((peer_id, msg, res_tx)) => {
-                                error!(security_log(security_events::INVALID_NETWORK_EVENT_MP)
-                                    .data("message", &msg)
-                                    .data("peer_id", &peer_id)
+                                error!(
```

Comment:
I think this logging level is probably dangerous, couldn't a public fn send me a bunch of invalid messages causing me to overwhelm my logging infra? Is this interesting?

Reply:
yes potentially, I didn't really think of attacks through logs, but we've been talking about the state sync log that can log pretty large messages and it seems to be true for a lot of these. The problem if we don't log the message is that we have no proofs that the other validator is misbehaving. As these are enabled not just on validators we could just enable most of these logs only if we are a validator (but on both the validator network and the full node network). Does that make sense?